core/ipc/rpc/api/src/main/java/org/opennms/core/rpc/api/RpcClientFactory.java

```diff
@@ -35,7 +35,14 @@
  */
 public interface RpcClientFactory {
 
-    static final String LOG_PREFIX = "ipc";
+    // RPC Metrics related constants.
+    public static final String JMX_DOMAIN_RPC = "org.opennms.core.ipc.rpc";
```

Comment:
We can omit the `public` and `static` qualifiers here since we're in an interface.